test/e2e/apimachinery/protocol.go

```diff
@@ -50,11 +50,13 @@ var _ = SIGDescribe("client-go should negotiate", func() {
 			cfg.AcceptContentTypes = accept
 
 			c := kubernetes.NewForConfigOrDie(cfg)
-			svcs, err := c.CoreV1().Services("default").Get(context.TODO(), "kubernetes", metav1.GetOptions{})
+			ns := f.Namespace.Name
+			before, err := c.CoreV1().ConfigMaps(ns).List(context.TODO(), metav1.ListOptions{})
 			framework.ExpectNoError(err)
-			rv, err := strconv.Atoi(svcs.ResourceVersion)
+			name := strings.NewReplacer("/", "-", ",", "-").Replace(s)
```

Comment:
can we use the range index to make a simple configmap name instead of naming it `application-json-application-vnd.kubernetes.protobuf`?